pkg/chaosdaemon/util.go

```diff
@@ -120,3 +122,29 @@ func withNetNS(ctx context.Context, nsPath string, cmd string, args ...string) *
 
 	return exec.CommandContext(ctx, "nsenter", args...)
 }
+
+// ContainerKillByContainerID kills container according to container id
+func (c DockerClient) ContainerKillByContainerID(ctx context.Context, containerID string) error {
+	err := c.client.ContainerKill(ctx, containerID[strings.Index(containerID, "//")+2:], "SIGKILL")
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// ContainerKillByContainerID kills container according to container id
+func (c ContainerdClient) ContainerKillByContainerID(ctx context.Context, containerID string) error {
+	containerID = containerID[len(containerdProtocolPrefix):]
+	container, err := c.client.LoadContainer(ctx, containerID)
+	if err != nil {
+		return err
+	}
+
+	err = container.Delete(ctx)
```

Comment:
I think maybe we need to add the deleteOption `containerd.WithSnapshotCleanup` here. I am not familiar with containerd,  but I check the implementation in [container cri](https://github.com/containerd/cri/blob/c955404ce7346197ba6286c7e487d9e9bdd0c084/pkg/server/container_remove.go#L66)  and [ctr](https://github.com/containerd/containerd/blob/4f348c5b5b53a70b282ab5b81b2cc8f03bf6a338/cmd/ctr/commands/containers/containers.go#L168), they all have this option. I am not sure whether this is needed.

Also, I am not sure whether this is equivalent with sending `SIGKILL` to that container or not?  There is already an implementation to do kill to the task, check [here](https://github.com/containerd/containerd/blob/4f348c5b5b53a70b282ab5b81b2cc8f03bf6a338/cmd/ctr/commands/tasks/kill.go#L28)